pkg/jobs/workers_list.go

```diff
@@ -59,6 +59,17 @@ func GetWorkersList() ([]*WorkerConfig, error) {
 	return workers, nil
 }
 
+// GetWorkersNamesList returns the names of the configured workers
+func GetWorkersNamesList() []string {
+	workers, _ := GetWorkersList()
+	defaultWorkers := make([]string, len(workers))
+
+	for _, w := range workers {
+		defaultWorkers = append(defaultWorkers, w.WorkerType)
+	}
```

Comment:
Don't you get a slice with twice the number of expected strings, and the first half with empty strings?

I'd have written the loop as:

```go
	for i, w := range workers {
		defaultWorkers[i] = w.WorkerType
	}
```